### PR TITLE
Add missing info in Message help page

### PR DESCRIPTION
**Pull request by phutanu**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-instrument-addon-custom-device/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?
Update Message Help page with "Value Change & Trigger" generation type info.

### Why should this Pull Request be merged?
In order to provide description for the Instrument Addon users regarding this message generation type.

### What testing has been done?
The Page was compiled in the .chm file and checked in System Explorer to make sure it works and displays well.

### CODE REVIEW

File: Source/Addon/Support Files/Help/HTML Help Source/Message.html [hunk 1]
```diff
@@ -15,15 +15,37 @@
...
-<p>Select the generation type from the <strong>Generation Type</strong> field.  <strong>Trigger</strong> will be the most common.  If you are sellecting <strong>Periodic & Trigger</strong>, set the <strong>Period (ms)</strong> to the desired setting.</p>
-<p>Note: Limit the number of periodic messages as much as possible on slower protocols and baud rates as this could delay other messages.</p>
-<h3>Automatically Include Message Protection</h3>
-<p>Message protections are often used to make sure the data sent and received match.  Common protection schemes include checksums or CRCs.  To enable this, check the <strong>Include Protection</strong> field and select the desired type.  The protection will be automatically calculated for you and placed at the end of the message.</p>
+   <p class="Body">
+   This section contains the items for defining a message.  If the message is a write only, configure <strong>Command</strong> and delete the <strong>Response</strong>.  If the message is a read only, configure the <strong>Response</strong> and delete the <strong>Command</strong>.  If the message is a query (write then read) configure both.
```
Comment (bogdanp-ni) [inline Source/Addon/Support Files/Help/HTML Help Source/Message.html:19]: ... configure **the** Command...

Comment (bogdanp-ni) [inline Source/Addon/Support Files/Help/HTML Help Source/Message.html:19]: ...query (write then read) configure both -> add a comma after the closing parenthesis.

File: Source/Addon/Support Files/Help/HTML Help Source/Message.html [hunk 2]
```diff
@@ -15,15 +15,37 @@
...
+            <strong>Trigger Only</strong> fits for single event messages.  As an example, setting the instrument configuration at the beginning of a test.
+         </p>
+         <p>
+            <strong>Periodic & Trigger</strong> has the same triggerable function as Trigger Only, but will automatically execute the message after a set period.  So, if you have a period of 1 second and don't trigger the message, it will automatically execute by itself.  If you trigger the message, the periodic timers get reset.
+         </p>
+         <p>
+            Note: The starting and stopping of the periodic generation can be controlled through the Enable Periodic channel.
+         </p>
+         <p>
+            <strong>Value Change & Trigger</strong> will trigger the message by using the Trigger Channel or any time a VeriStand channel used in the command field is changed.  If the command message contains multiple VeriStand channels, the message will be sent each time one of the channels has changed.
```
Comment (bogdanp-ni) [inline Source/Addon/Support Files/Help/HTML Help Source/Message.html:35]: "any time" -> anytime

File: Source/Addon/Support Files/Help/HTML Help Source/Message.html (hunk 1 above)
Comment (phutanu) [reply Source/Addon/Support Files/Help/HTML Help Source/Message.html:19]: Yeah, I did not touched this part. I trusted the natives on this one. :)

Comment (phutanu) [reply Source/Addon/Support Files/Help/HTML Help Source/Message.html:19]: Yeah, I did not touched this part. I trusted the natives on this one. :)

Comment (phutanu) [reply Source/Addon/Support Files/Help/HTML Help Source/Message.html]: Change implemented.

Comment (phutanu) [reply Source/Addon/Support Files/Help/HTML Help Source/Message.html]: Change implemented.

File: Source/Addon/Support Files/Help/HTML Help Source/Message.html (hunk 2 above)
Comment (phutanu) [reply Source/Addon/Support Files/Help/HTML Help Source/Message.html]: Change Implemented.

**Pull request merged**